health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -441,47 +469,39 @@ const CreateChecklist = () => {
   };
 
 
-  const onSubmit = async (formData, flag = 0, preview = null) => {
+  const onSubmit = async (formData, flag = 0, preview = null, translations) => {
     let payload;
     if (flag === 1) {
       payload = payloadData(preview);
     } else {
       payload = payloadData(formData?.createQuestion?.questionData);
     }
+    let allLocalisations = [...uniqueLocal, ...translations].filter(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'allLocalisations' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)